README.rst

```diff
@@ -665,6 +693,11 @@ light blue: no ETA); as demonstrated below.
 |Screenshot-Jupyter2|
 |Screenshot-Jupyter3|
 
+Since v4.23.0, tqdm_notebook now can respect ``ncols`` and ``dynamic_ncols`` if 
+their values are given. Here is an example:
+
+|Screenshot-Jupyter4|
```

Comment:
seems a bit much to add an extra screen shot just for this. would be nicer to include `ncols` in one of the examples above

Reply:
Eh, the problem is I don't have gif generator... Or how about we just mention it and do not give an example?